crates/sui-protocol-config/src/lib.rs

```diff
@@ -140,6 +140,10 @@ struct FeatureFlags {
     // in end of epoch checkpoint proposals
     #[serde(skip_serializing_if = "is_false")]
     commit_root_state_digest: bool,
+    // If true, we are in rollout phase for the above feature,
+    // and thus should not be enabled on mainnet.
+    #[serde(skip_serializing_if = "is_false")]
+    commit_root_state_digest_rollout: bool,
```

Comment:
I think if we want to do this approach instead of having explicit per-chain permissions, we should use an enum instead of multiple bools, for instance

```
pub enum FeatureFlagValue {
   Disabled,
   Testing,
   Enabled
}
```